README.md

```diff
@@ -17,6 +17,11 @@ Assume all code executes in strict mode.
 ## Question 1
 
 Please list the JavaScript primitives and give one example of each.
+Number  5
+String  'This is a string'
+Boolean  true
+null     let value = null;
+undefined    let name;
```

Comment:
`undefined = undefined` When you do `let name` you are declaring the variable, just not assigning it yet.